megamek/src/megamek/client/ui/Messages.java

```diff
@@ -1,16 +1,16 @@
-/*
- * MegaMek - Copyright (C) 2005 Ben Mazur (bmazur@sev.org)
- *
- *  This program is free software; you can redistribute it and/or modify it
- *  under the terms of the GNU General Public License as published by the Free
- *  Software Foundation; either version 2 of the License, or (at your option)
- *  any later version.
- *
- *  This program is distributed in the hope that it will be useful, but
- *  WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
- *  or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
- *  for more details.
- */
+/*  
+* MegaMek - Copyright (C) 2020 - The MegaMek Team  
```

Comment:
Why are you removing the original side of things?

Reply:
Please explain. Im not remotely an expert on copyright in software applications so I thought why not update it to the current example in the wiki.

Reply:
I did that originally, but was told to instead leave the original and add the new line below it when I first started contributing. Hence the question.

Reply:
Basically we can't erase Ben's copyright on his work. Unless this is a total rewrite.